posts/2020-09-29-microprofile-openapi-beta-200011.adoc

```diff
@@ -0,0 +1,253 @@
+
+[source, java]
+----
+@APIResponseSchema(MyResponseObject.class)
+----
+
+[#mpopenapischeme]
+==== mp.openapi.schema.*
+The mp.openapi.schema.* MicroProfile Config property prefix has been added to allow you to define the schema for Java classes using configuration rather than code. The use of this property is functionally equivalent to the use of the @Schema annotation on a Java class, but may be used in cases where you do not have access to the source code of a class. The remainder of the property key must be the fully-qualified class name and the value must be a valid OpenAPI schema object, specified in the JSON format.
+For example, in the case where an application needs to represent dates in epoch milliseconds, the following configuration could be used (line escapes and indentation added for readability):
+
+[source, java]
+----
+mp.openapi.schema.java.util.Date = { \
+   "name": "EpochMillis" \
+   "type": "number", \
+   "format": "int64", \
+   "description": "Milliseconds since January 1, 1970, 00:00:00 GMT" \
+ }
+----
+
+Enable OpenAPI 2.0 in the server.xml, along with any other features you’re using.
+
+[source, xml]
+----
+<featureManager>
+       <feature>mpOpenAPI-2.0</feature>
+   </featureManager>
+----
+
+For more information:
+
+* link:{url-prefix}/guides/microprofile-openapi.html[Guide: Documenting RESTful APIs]
+* link:https://download.eclipse.org/microprofile/microprofile-open-api-2.0-RC3/apidocs/[Javadoc]
+* link:https://download.eclipse.org/microprofile/microprofile-open-api-2.0-RC3/microprofile-openapi-spec.html#release_notes_20[Release notes]
+
+=== Try it now
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 14, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>20.0.0.11-beta</version>
+  <type>pom</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[20.0.0.11-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+image::img/blog/logos/Jakarta_EE_logo.png[ Jakarta EE Logo,width=70%,align="center"]
+
+The main change you should be aware of in Jakarta EE 9 planned release is the names of packages changing to accomodate the new `jakarta.*` namespace. In this Open Liberty beta, we have more Jakarta EE 9 features with their name change completed. In this release most features had improvments, including many moving up to the latest API Release Candidates.
```

Comment:
Remove "planned release ". (and do the same in the template if necessary)

Reply:
"In this Open Liberty beta, we have more Jakarta EE 9 features with their name change completed. In this release most features had improvments, including many moving up to the latest API Release Candidates." -> "In this Open Liberty beta, we have more Jakarta EE 9 features with their name change completed. Most features have improvements too, including many moving up to the latest API Release Candidates."

Reply:
Done

Reply:
Updated in template as well.